lab1.js

```diff
@@ -75,7 +76,27 @@ var sentence2 = 'Come over here so you can scratch my belly.';
  HINT: the 'split' method on String will be useful.
 */
 
-// your code goes here
+function splitString(stringToSplit, separator) {
+  var arrayOfStrings = stringToSplit.split(separator);
+  for (var i = 0; i < arrayOfStrings.length; i++) {
+    arrayOfStrings[i] = 'chirp';
+  }
+  // console.log(arrayOfStrings);
```

Comment:
you're not returning anything out of this function right now. arrayOfStrings is scoped inside this function; joinArray can't see this variable.